backend/database/migrations/2021_01_27_213544_insert_user_roles.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+use App\Models\Role;
+
+class InsertUserRoles extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $roles = Role::getArrayOfAllRoleNames();
```

Comment:
I know it's less DRY, but the roles to be created should be explicit here rather than pulled from the Roles class.

Reply:
Done. I removed the dependence on the `Role` class here.